packages/storybook/stories/components/form-field/documentation.mdx

```diff
@@ -0,0 +1,51 @@
+import { Canvas, Meta, Markdown } from '@storybook/blocks';
+import SpecificationsFormField from '@ovhcloud/ods-components/src/components/form-field/documentation/spec.md?raw';
+import { Banner } from '../../banner';
+import { DocNavigator } from '../../doc-navigator';
+import * as FormFieldStories from './form-field.stories';
+
+<Meta of={ FormFieldStories } name='Documentation' />
+
+<Banner of={ FormFieldStories } />
+
+# Overview
+
+A Form Field component is used to wrap several form components with logic, visual hints and additional styling:
+
+<Canvas of={ FormFieldStories.Default } sourceState='none' />
+
+<DocNavigator of={ FormFieldStories } />
+
+<Markdown>{ SpecificationsFormField }</Markdown>
+
+# Style customization
+
+You can add your own style on the form-field elements using `::part` or by directly updating the slot elements.
```

Comment:
feels weired to mention parts here as they can use whatever they want in the form-field, not always ods with parts